refarch-backend/src/main/java/de/muenchen/refarch/common/exception/NotFoundException.java

```diff
@@ -0,0 +1,17 @@
+package de.muenchen.refarch.common.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+/** Exception if data cannot be found. */
+@SuppressWarnings("PMD.MissingSerialVersionUID")
```

Comment:
> This way, we wouldn't need Serializable

Don't think that true, as i had to add the serial version to all my custom exceptions too. But why not just adding it instead of the SuppressWarning?

Reply:
That's an option as well. I guess the serial Version has to be unique for each class implementing Serializable, right? How can we enforce the use of unique UIDs?

Reply:
No as far as i know it only needs to be increased when modifying the class.
As example i set everyone to
```
@Serial
private static final long serialVersionUID = 1L;
```

Reply:
I think the solution of @DanielOber is fine here. Can we resolve this discussion?